src/test/java/unit/BqDatasetNumTables.java

```diff
@@ -24,8 +24,8 @@
   // external dataset to use for creating BQ dataset references in the workspace
   private DatasetReference externalDataset;
 
-  // name of table in external dataset
-  private String externalDatasetTableName = "testTable";
+  // name of tables in external dataset
+  private String sharedExternalTable = "testTable2";
```

Comment:
nit: does "shared" mean anything specific here? or is this just leftover from a test restructuring?